### ISSUE TITLE
Comment is removed from expression in test.each template literal

### ISSUE THREAD

**Issue by ejones** [prettier/prettier#9147]
**Prettier 2.1.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuE8DOMB0cCGYAWABgDpQAEZAZhBBWQD5kBGOATqRQCTDEi8C+DMt16ioAegBUZHGUgBbeQhhlJ4-qUIAKUSAA0ZLQEoyAXgB8ZYPyMBufSAgAHGAEtoaZKDasIAdwAFNgRPFBwAGz8cAE9PPRAmVjwAazgYAGUnPFcoAHNkGFYAVzh4uHkmOAATKuqAGRw8opxcuAAxCFZ5HBg3POQQHCKYCAd8GHlwgHV8V3QssDh0kLnXADc56IGwNDiQHLQ4VhgApNzu5EoIw-iAKzQADwAhJLBUjJwlOpy4S+vSkD3B7pHK5cJwACKRQg8D+4RuICyrEOrAGLEq4QcTlYORgU1cVRg+GQAA4AAzxbEQQ5TJJOAbYuAota-eIAR2h8FOzlCgzQAFooHBqtUHKw4BzXOLTi0LkgrvCAYd5K4CsUlaDwVCYb95f94jAcEx8YTiUgAEwGpKucKggDCEEUOAGTIArA4iocACpG0IKhFrEoASSgtVg6TAOJcAEFQ+kYNFwXDDvx+EA)

**Input:**

```js
test.each`
  foo   | bar
  ${""} | ${""
/* a comment */}
`("", () => {});
```

**Output:**

```js
test.each`
  foo   | bar
  ${""} | ${""}
`("", () => {});
```

**Expected behavior:**
The comment should be preserved in the output.

Note: it looks like this only happens if the comment is on a new line. If the comment starts on the same line as the template expression, it is preserved - [example 1](https://prettier.io/playground/#N4Igxg9gdgLgprEAuE8DOMB0cCGYAWABgDpQAEZAZhBBWQD5kBGOATqRQCTDEi8C+DMt168yAegBUZHGUgBbeQhhlJ4-qUIAKUSAA0ZLQEoyAXgB8ZYPyMBufSAgAHGAEtoaZKDasIAdwAFNgRPFBwAGz8cAE9PPRAmVjwAazgYAGUnPFcoAHNkGFYAVzh4uHkmOAATKuqAGRw8opxcuAAxCFZ5HBg3POQQHCKYCAd8GHlwgHV8V3QssDh0kLnXADc56IGwNDiQHLQ4VhgApNzu5EoIw-iAKzQADwAhJLBUjJwlOpy4S+vSkD3B7pHK5cJwACKRQg8D+4RuICyrEOrAGLEq4QcTlYORgU1cVRg+GQAA4AAzxbEQQ5TJJOAbYuAota-eIAR2h8FOzlCgzQAFooHBqtUHKw4BzXOLTi0LkgrvCAYd5K4CsUlaDwVCYb95f94jAcEx8YTiUgAEwGpKucKggDCEEUOAGTIArA4iocACpG0IKhFrEoASSgtVg6TAOJcAEFQ+kYNFwXDDvx+EA), [example 2](https://prettier.io/playground/#N4Igxg9gdgLgprEAuE8DOMB0cCGYAWABgDpQAEZAZhBBWQD5kBGOATqRQCTDEi8C+DMt168yAegBUHMjjKQAtgoQwyk8f1KEAFKJAAaMtoCUZALwA+MsH7GA3AZAQADjACW0NMlBtWEAO4ACmwIXig4ADb+OACeXvogTKx4ANZwMADKznhuUADmyDCsAK5wCXAKTHAAJtU1ADI4+cU4eXAAYhCsCjgw7vnIIDjFMBCO+DAKEQDq+G7o2WBwGaHzbgBu8zGDYGjxILlocKwwgcl5PciUkUcJAFZoAB4AQslgaZk4yvW5cFc3ZRAD0eGVyeQicAAisUIPB-hFbiBsqwjqxBiwqhFHM5WLkYNM3NUYPhkAAOAAMCRxECO02SzkGOLgqPWfwSAEcYfAzi4wkM0ABaKBwGo1RysOCctwSs6tS5Ia4IwFHBRuQolZVgiHQ2F-BUAhIwHBMAlEklIABMhuSbgiYIAwhAlDhBsyAKyOYpHAAqxrCisR61KAEkoHVYBkwLjXABBMMZGAxCHwo78fhAA).

**Comment by agamkrbit** [comment]
can I pick this?

**Comment by fisker** [comment]
Go ahead.

**Comment by SuperITMan** [comment]
Hi everyone!
Would you have any news about this issue? 😊

**Comment by thorn0** [comment]
Fixed by #9703
